subsys/bluetooth/audio/mpl.c

```diff
@@ -1740,68 +1740,68 @@ void playing_state_command_handler(struct mpl_cmd command,
 				pl.track_pos = 0;
 				media_proxy_pl_track_position_cb(pl.track_pos);
 			}
-			ntf.result_code = BT_MCS_OPC_NTF_SUCCESS;
+			ntf->result_code = BT_MCS_OPC_NTF_SUCCESS;
 		} else {
-			ntf.result_code = BT_MCS_OPC_NTF_CANNOT_BE_COMPLETED;
+			ntf->result_code = BT_MCS_OPC_NTF_CANNOT_BE_COMPLETED;
 		}
 		media_proxy_pl_command_cb(ntf);
 		break;
 	case BT_MCS_OPC_PREV_GROUP:
 		do_full_prev_group(&pl);
-		ntf.result_code = BT_MCS_OPC_NTF_SUCCESS;
+		ntf->result_code = BT_MCS_OPC_NTF_SUCCESS;
 		media_proxy_pl_command_cb(ntf);
 		break;
 	case BT_MCS_OPC_NEXT_GROUP:
 		do_full_next_group(&pl);
-		ntf.result_code = BT_MCS_OPC_NTF_SUCCESS;
+		ntf->result_code = BT_MCS_OPC_NTF_SUCCESS;
 		media_proxy_pl_command_cb(ntf);
 		break;
 	case BT_MCS_OPC_FIRST_GROUP:
 		do_full_first_group(&pl);
-		ntf.result_code = BT_MCS_OPC_NTF_SUCCESS;
+		ntf->result_code = BT_MCS_OPC_NTF_SUCCESS;
 		media_proxy_pl_command_cb(ntf);
 		break;
 	case BT_MCS_OPC_LAST_GROUP:
 		do_full_last_group(&pl);
-		ntf.result_code = BT_MCS_OPC_NTF_SUCCESS;
+		ntf->result_code = BT_MCS_OPC_NTF_SUCCESS;
 		media_proxy_pl_command_cb(ntf);
 		break;
 	case BT_MCS_OPC_GOTO_GROUP:
-		if (command.use_param) {
-			do_full_goto_group(&pl, command.param);
-			ntf.result_code = BT_MCS_OPC_NTF_SUCCESS;
+		if (command->use_param) {
+			do_full_goto_group(&pl, command->param);
+			ntf->result_code = BT_MCS_OPC_NTF_SUCCESS;
 		} else {
-			ntf.result_code = BT_MCS_OPC_NTF_CANNOT_BE_COMPLETED;
+			ntf->result_code = BT_MCS_OPC_NTF_CANNOT_BE_COMPLETED;
 		}
 		media_proxy_pl_command_cb(ntf);
 		break;
 	default:
-		BT_DBG("Invalid command: %d", command.opcode);
-		ntf.result_code = BT_MCS_OPC_NTF_NOT_SUPPORTED;
+		BT_DBG("Invalid command: %d", command->opcode);
+		ntf->result_code = BT_MCS_OPC_NTF_NOT_SUPPORTED;
```

Comment:
Unrelated to the changes in this PR but it looks like we always override `ntf->result_code` before calling `media_proxy_pl_command_cb`. If that is the case, why do we even use a `mpl_cmd_ntf` as a parameter instead of just the `requested_opcode`, and then just create the `mpl_cmd_ntf` in the command handlers?

Reply:
It is not over-ridden, it is set.
But yes, could do as you suggest.  It adds one more variable (opcode + `mpl_cmd_ntf` instead of just the last one), but maybe it looks cleaner.
I believe the thought was that the notification struct must be filled in with values from two places in the code, so it was just as easy to send it along.
No strong opinion on this one.

Reply:
> It adds one more variable (opcode + mpl_cmd_ntf instead of just the last one),

If we send the opcode by parameter, we don't need the `mpl_cmd_ntf` argument at all, do we? 

mpl_cmd_ntf contains the opcode and a result code; if we get the opcode from the caller, and the result code from this function, this function can just allocate the `mpl_cmd_ntf` struct, right? 

It is outside the scope of this PR, so do with it as you will :)